libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -28,7 +28,10 @@ extern const AP_HAL::HAL& hal;
 */
 AP_RangeFinder_LightWareI2C::AP_RangeFinder_LightWareI2C(RangeFinder::RangeFinder_State &_state, AP_HAL::OwnPtr<AP_HAL::I2CDevice> dev)
     : AP_RangeFinder_Backend(_state)
-    , _dev(std::move(dev)) {}
+    , _dev(std::move(dev))
+{
+    _instanceState = _state;
```

Comment:
State is already saved in the backend...

Reply:
I will change.